assets/js/admin/form-builder.js

```diff
@@ -222,6 +223,290 @@
+
+			// Initialize the field as an IonRangeSlider field.
+			$field.find( '.evf-range-slider-preview' ).ionRangeSlider();
+
+			// Slider handle/highlight/track color change handler.
+			EVFPanelBuilder.initializeSliderHandleColorOption( field_id );
+			EVFPanelBuilder.initializeSliderHighlightColorOption( field_id );
+			EVFPanelBuilder.initializeSliderTrackColorOption( field_id );
+			EVFPanelBuilder.updateRangeSliderBasicOptions( field_id );
+			EVFPanelBuilder.updateRangeSliderColors( field_id );
+		},
+
+		/**
+		 * Initialize Range Slider Handle Color Picker.
+		 *
+		 * @since 1.7.0
+		 */
+		initializeSliderHandleColorOption: function( field_id ) {
+			var $field_options_container = $( '#everest-forms-field-option-' + field_id );
+
+			// Initialize color picker for Handle Color option.
+			$field_options_container.find( '.evf-range-slider-handle-color' )
+			.wpColorPicker({
+				change: function( event, ui ) {
+					var new_color = $( event.target ).val();
+					var field_id = $( this ).closest( '.everest-forms-field-option-row' ).data( 'field-id' );
+					var current_skin = $field_options_container.find( '.evf-range-slider-skin' ).val();
+
+					EVFPanelBuilder.setSliderHandleColor( field_id, new_color, current_skin );
+				}
+			});
+		},
+
+		/**
+		 * Initialize Range Slider Highlight Color Picker.
+		 *
+		 * @since 1.7.0
+		 */
+		initializeSliderHighlightColorOption: function( field_id ) {
+			var $field = $( '#everest-forms-field-' + field_id );
+			var $field_options_container = $( '#everest-forms-field-option-' + field_id );
+
+			$field_options_container.find( '.evf-range-slider-highlight-color' )
+			.wpColorPicker({
+				change: function( event, ui ) {
+					var new_color = $( event.target ).val();
+
+					if ( new_color ) {
+						$field.find( '.irs-bar' ).css( 'background', new_color );
+					}
+				}
+			});
+		},
+
+		/**
+		 * Initialize Range Slider Track Color Picker.
+		 *
+		 * @since 1.7.0
+		 */
+		initializeSliderTrackColorOption: function( field_id ) {
+			var $field = $( '#everest-forms-field-' + field_id );
+			var $field_options_container = $( '#everest-forms-field-option-' + field_id );
+
+			$field_options_container.find( '.evf-range-slider-track-color' )
+			.wpColorPicker({
+				change: function( event, ui ) {
+					var new_color = $( event.target ).val();
+
+					if ( new_color ) {
+						$field.find( '.irs-line' ).css( 'background', new_color );
+					}
+				}
+			});
+		},
+
+		/**
+		 * Update a Range Slider field's Handle/Highlight/Track colors.
+		 *
+		 * @since 1.7.0
+		 */
+		updateRangeSliderColors: function ( field_id ) {
+			var $field = $( '#everest-forms-field-' + field_id );
+			var $field_options_container = $( '#everest-forms-field-option-' + field_id );
+
+			// Set handle color for the Slider field.
+			var handle_color = $field_options_container.find( '.evf-range-slider-handle-color' ).val();
+			var skin = $field_options_container.find( '.evf-range-slider-skin' ).val();
```

Comment:
Comment here is redundant, as you've named the variable quite nicely. Initializing all of them in a single statement is smart.

Reply:
Resolved with 38e1bd4fdd984eab4133668204c60ca1ca725b73